Packaging/rg35xx_garlic/skeleton_GarlicOS/ROMS/PORTS/Diablo/.local/share/diasurgical/devilution/diablo.ini

```diff
@@ -0,0 +1,221 @@
+[Language]
```

Comment:
This is going to get out of sync very quickly. Why is this file needed?
Our general approach for platforms that need different defaults is to set the defaults via CMake variables => `#define`.